[DX-1438] Update installation.md (#4827)
TykTechnologies/tyk-docs: tyk-docs/content/getting-started/installation.md

--- 
date: 2017-03-08T18:15:30+13:00

> title: Guide to tyk installations

Author: ```suggestion
title: Guide to Tyk Installations
```

tags: ["installing", "installation", "Tyk Installation Options", “Guide to Tyk installations”, "Tyk Open Source API Gateway", "Tyk Self-Managed Installation", "Tyk Cloud Installation"]
description: "This guide helps you choose the Tyk installation that best suits your needs."
aliases:
    - /try-out-tyk/tutorials/tutorials/
---

> Welcome to the Tyk Installation Guide! Whether you're new to Tyk or an experienced user, this guide will help you find the installation option that best meets your needs. Let's get started!

Author: ```suggestion
Welcome to the Tyk Installation Guide! Whether you're new to Tyk or an experienced user, this guide will help you find the installation model that best meets your needs. Let's get started!
```

<br/>

{{< button_left href="tyk-self-managed/install" color="green" content="Install Self-Managed" >}}

### Tyk Cloud

Get up and running quickly with our fully managed *Tyk Cloud* SaaS.
A fully managed service that makes it easy for API teams to create, secure, publish and maintain APIs at any scale, anywhere in the world.

**Includes: Tyk API Gateway, Tyk Dashboard, Tyk Portal, Tyk UDG**

<br/>

{{< button_left href="tyk-cloud/getting-started" color="green" content="Deploy Tyk Cloud" >}}

### Tyk Hybrid

Combine the control of self-managed with the convenience of cloud using our hybrid deployment. The control plane is managed by Tyk (on *Tyk Cloud*) and the data plane is installed and managed by you.

**Includes: Tyk API Gateway, Tyk Dashboard, Tyk Portal, Tyk UDG**

<br/>

{{< button_left href="tyk-cloud/environments-deployments/hybrid-gateways" color="green" content="Install Tyk Hybrid" >}}

 
### Open source Tyk Gateway (OSS)

Install and manage Tyk yourself with our open-source API Gateway.

**Includes: Tyk OSS Gateway**

<br/>

{{< button_left href="apim/open-source/installation" color="green" content="Install Tyk OSS" >}}

---

## Not Sure Which to Choose?

> Visit our [Comparison Page](https://tyk.io/about/contact) to understand the different installation types and their benefits.

Author: ```suggestion
Visit our [Comparison Page]({{< ref "/apim" >}}) to understand the different installation types and their benefits.
```
